prime-angular-frontend/src/app/modules/adjudication/shared/components/enrollee-table/enrollee-table.component.ts

```diff
@@ -129,17 +137,25 @@ export class EnrolleeTableComponent implements OnInit {
       appliedDateRangeEnd: '',
       renewalDateRangeStart: '',
       renewalDateRangeEnd: '',
+      assignedTo: ''
     });
   }
 
   private initForm() {
+    this.authService.getAdmin$()
+      .subscribe(({ idir }: Admin) => this.adjudicatorIdir = idir);
+
     this.dataSource.filterPredicate = this.getFilterPredicate();
 
     this.form.valueChanges.subscribe(value => {
       const filter = { ...value, name: value.name } as string;
```

Comment:
Can you also remove `as string` we're passing an object literal, and shouldn't be saying it is something else entirely to make it passed the type system.  You'll also want to add `filter: any` to `getFilterPredicate`.